source/stylesheets/refills/unstyled/_vertical-tabs.scss

```diff
@@ -0,0 +1,53 @@
+.vertical-tabs-container {
+  $large-screen: em(640) !default;
+  $vertical-tabs-height: em(100);
+
+  @include clearfix;
+
+  .vertical-tabs {
+    display: none;
+
+    @include media($large-screen) {
+      display: inline;
+      float: left;
+      height: $vertical-tabs-height;
+      width: 20%;
+    }
+  }
+
+  .vertical-tab {
+    @include media($large-screen) {
+      display: block;
+
+      &.is-active {
+        color: #ff0000;
```

Comment:
He’s wrong. Something got added to the linter that shouldn't have and needs to be fixed.